lib/eventemitter2.js

```diff
@@ -670,15 +660,14 @@
         leaf._listeners = null;
       }
     }
-    else {
-      if (!this._events || !this._events[type]) return this;
+    else if (this._events) {
```

Comment:
i think it's more clear way to remove listeners
if there is some issues with it i can rollback it